capstone/src/main/webapp/form-loader.js

```diff
@@ -13,26 +15,14 @@ function callScriptFunction() {
         "devMode": true
     }
     }).then(function(resp) {
-        var result = resp.result;
-        if (result.error && result.error.status) {
+        if (resp.result?.error?.status) {
             // The API encountered a problem before the script started executing
-            appendPre('Error calling API:');
-            appendPre(JSON.stringify(result, null, 2));
-        } else if (result.error) {
+            console.log('Error calling API: ' + result);
+        } else if (resp.result?.error) {
```

Comment:
This is the question I had in standup. I believe this is the correct way of error handling and checking if the error is undefined!

Reply:
This is okay and pretty common, a slightly safer way would be:

`resp.result?.error != null`

In general you should always use `===` in JS, but when comparing to null this is an exception. A good write up:
http://adripofjavascript.com/blog/drips/equals-equals-null-in-javascript.html

Reply:
Sounds good! I just implemented this fix and should have this updated now.